portfolio/src/main/webapp/index.html

```diff
@@ -74,17 +74,11 @@
               />
               <small class="d-none" id="working-hour-warning"></small>
             </li>
-            <li class="nav-item dropdown">
-              <a
-                class="nav-link dropdown-toggle"
-                role="button"
-                data-toggle="dropdown"
-                aria-haspopup="true"
-                aria-expanded="false"
-              >
-                Scheduling algorithm preferences
-              </a>
-              <div class="dropdown-menu" aria-labelledby="navbar-dropdown"></div>
+            <li class="nav-item nav-link">
+              Scheduling Type:
+              <select id="algorithm-type" name="algorithm-type">
+                <option value="SHORTEST_TASK_FIRST">Shortest tasks first</option>
```

Comment:
This is okay, but I think the default option should be highest priority tasks first. Consider adding TODO comment.

Reply:
Once other algorithms are implemented they'll be added and we'll change the default option.